backends-common/postgres/src/test/java/org/apache/james/backends/postgres/PostgresFixture.java

```diff
@@ -94,5 +94,6 @@ public String schema() {
         .withDatabaseName(DEFAULT_DATABASE.dbName())
         .withUsername(DEFAULT_DATABASE.dbUser())
         .withPassword(DEFAULT_DATABASE.dbPassword())
-        .withCreateContainerCmdModifier(cmd -> cmd.withName("james-postgres-test-" + UUID.randomUUID()));
+        .withCreateContainerCmdModifier(cmd -> cmd.withName("james-postgres-test-" + UUID.randomUUID()))
+        .withCommand("postgres", "-c", "max_connections=300");
```

Comment:
Q: This command takes effect immediately, without needing to restart postgres?

Reply:
restart postgres -> what do you mean?

Reply:
Does Postgres server need to restart/reload after  setting `max_connections=300` ?

Reply:
No